v22.2/alter-table.md

```diff
@@ -9,36 +9,2977 @@ docs_area: reference.sql
+{{site.data.alerts.end}}
+
+By default, `DROP COLUMN` drops any [indexes](indexes.html) on the column being dropped, and any indexes that reference the column, including [partial indexes](partial-indexes.html) with predicates that reference the column and indexes with [`STORING` clauses](create-index.html#store-columns) that reference the column.
+
+For examples, see [Drop columns](#drop-columns).
+
+#### Required privileges
+
+The user must have the `CREATE` [privilege](security-reference/authorization.html#managing-privileges) on the table.
+
+#### Parameters
+
+Parameter | Description |
+----------|-------------|
+`column_name` | The name of the column you want to drop.<br><br>When a column with a [`CHECK`](check.html) constraint is dropped, the `CHECK` constraint is also dropped.
+`CASCADE` | Drop the column even if objects (such as [views](views.html)) depend on it; drop the dependent objects, as well. `CASCADE` will drop a column with a foreign key constraint if it is the only column in the reference.<br><br>`CASCADE` does not list the objects it drops, so should be used cautiously.<br><br> `CASCADE` is not required to drop an indexed column, or a column that is referenced by an index. By default, `DROP COLUMN` drops any [indexes](indexes.html) on the column being dropped, and any indexes that reference the column, including [partial indexes](partial-indexes.html) with predicates that reference the column and indexes with [`STORING` clauses](create-index.html#store-columns) that reference the column.
+`RESTRICT` | *(Default)* Do not drop the column if any objects (such as [views](views.html)) depend on it.
+
+### `DROP CONSTRAINT`
+
+Use `ALTER TABLE ... DROP CONSTRAINT` to remove [`CHECK`](check.html) and [`FOREIGN KEY`](foreign-key.html) constraints from columns.
+
+[`PRIMARY KEY`](primary-key.html) constraints can be dropped with `DROP CONSTRAINT` if an [`ADD CONSTRAINT`](alter-table.html#add-constraint) statement follows the `DROP CONSTRAINT` statement in the same transaction.
+
+{{site.data.alerts.callout_success}}
+When you change a primary key with [`ALTER TABLE ... ALTER PRIMARY KEY`](alter-table.html#alter-primary-key), the old primary key index becomes a secondary index. If you do not want the old primary key to become a secondary index, use `DROP CONSTRAINT`/[`ADD CONSTRAINT`](alter-table.html#add-constraint) to change the primary key.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_info}}
+For information about removing other constraints, see [Constraints: Remove Constraints](constraints.html#remove-constraints).
+{{site.data.alerts.end}}
+
+For examples, see [Drop constraints](#drop-constraints).
+
+#### Required privileges
+
+The user must have the `CREATE` [privilege](security-reference/authorization.html#managing-privileges) on the table.
+
+#### Parameters
```

Comment:
I wonder if instead of including another diagram for each subcommand (which you mentioned as a future possibility) you could add a sentence to each parameter section directing the reader to the synopsis diagram at the top? Each parameter is called out in the overarching diagram? (I think...)

Reply:
Really good simple suggestion - I've added this to all the updated docs!